src/DynamicRupture/FrictionLaws/NoFault.h

```diff
@@ -17,14 +17,14 @@ class NoFault : public BaseFrictionLaw<NoFault> {
 
   void updateFrictionAndSlip(FaultStresses const& faultStresses,
                              TractionResults& tractionResults,
-                             std::array<real, misc::numPaddedPoints> const& stateVariableBuffer,
-                             std::array<real, misc::numPaddedPoints> const& strengthBuffer,
+                             std::array<real, misc::numPaddedPoints>& stateVariableBuffer,
+                             std::array<real, misc::numPaddedPoints>& strengthBuffer,
```

Comment:
Why remove the constness?

Reply:
That may have been the cause for the `const` confusion for `stateVariableBuffer` and so on—it's my guess so at least by now—because in all other cases, the `const` is also missing.